octopoes/nibbles/ask_network_questions/ask_network_questions.py

```diff
@@ -0,0 +1,18 @@
+import json
+from collections.abc import Iterator
+from pathlib import Path
+
+from octopoes.models import OOI
+from octopoes.models.ooi.network import Network
+from octopoes.models.ooi.question import Question
+
+
+def nibble(input_ooi: Network) -> Iterator[OOI]:
+    network = input_ooi
+    current_dir = Path(__file__).parent
+
+    # Find all question schema files in current directory
+    for schema_path in current_dir.glob("question_schema_*.json"):
+        with schema_path.open() as f:
+            schema = json.load(f)
```

Comment:
perhaps we should try/catch here to be extra safe?

Reply:
I think having this throw an Exception is good since its a programming mistake?

Reply:
True, I guess there's not really a way a corrupt file could appear in this directory that make "correct" schemas fail.